specs/data_structures.md

```diff
@@ -930,7 +930,12 @@ For explanation on entries, see the [reward distribution rationale document](../
 
 ### Decimal
 
-TODO define a format for numbers in the range `[0,1]`
+| name          | type   | description           |
+| ------------- | ------ | --------------------- |
+| `numerator`   | uint64 | Rational numerator.   |
+| `denominator` | uint64 | Rational denominator. |
+
+Represents a (potentially) non-integer number.
```

Comment:
This will be a relatively big change compared to the SDK. I guess switching from the current implementation to this rational type is really low priority, right?

Reply:
Right, it doesn't affect the devnet at all. Both Decimal as it currently exists in the SDK and Rational (i.e. our Decimal type) would produce similar numbers, so it wouldn't even change the state transitions in anything but in some cases that required lots of precision.

Reply:
Notes under  "optional to implement until we there is nothing left to crunch on" :-D  (tagging @evan-forbes s.t. he is also aware of this.